src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -1399,3 +1399,127 @@ describe('Click through all steps', () => {
         expect(store.getStore().getState().composes.allIds).toEqual(ids);
     });
 });
+
+describe('Keyboard accessibility', () => {
+    const setUp = async () => {
+        const view = renderWithReduxRouter(<CreateImageWizard />, {}, '/imagewizard');
+        history = view.history;
+    };
+
+    const clickNext = () => {
+        const next = screen.getByRole('button', { name: /Next/ });
+        next.click();
+    };
+
+    const selectAllEnvironments = () => {
+        const awsTile = screen.getByTestId('upload-aws');
+        awsTile.click();
+        const googleTile = screen.getByTestId('upload-google');
+        googleTile.click();
+        const azureTile = screen.getByTestId('upload-azure');
+        azureTile.click();
+        const virtualizationCheckbox = screen.getByRole('checkbox', {
+            name: /virtualization guest image checkbox/i
+        });
+        virtualizationCheckbox.click();
+    };
+
+    const fillAzureInputs = () => {
+        userEvent.type(screen.getByTestId('azure-tenant-id'), 'b8f86d22-4371-46ce-95e7-65c415f3b1e2');
+        userEvent.type(screen.getByTestId('azure-subscription-id'), '60631143-a7dc-4d15-988b-ba83f3c99711');
+        userEvent.type(screen.getByTestId('azure-resource-group'), 'testResourceGroup');
+    };
+
+    test('autofocus on each step first input element', async () => {
+        setUp();
+
+        // Details
+        const detailsInput = screen.getByRole('textbox', { name: /image name/i });
+        expect(detailsInput).toHaveFocus();
+        clickNext();
+
+        // Image output
+        // const releaseSelect = screen.getByRole('button', { name: /open menu/i });
```

Comment:
Can you remove the commented tests?

Reply:
Yes, definitely! I couldn't decide whether it was best to remove them or leave them as 'futures', to be un-commented when those steps do have autofocus. But they're so simple, it's probably better to just rewrite them when that time comes.

Reply:
I'd actually create an issue in github to track it. The code/tests might change by the time you would implement these so better to track the need for them at a higher level (issues) since you may need to implement them differently anyways.